src/dsp/fm_deemph.h

```diff
@@ -58,13 +58,15 @@ class fm_deemph : public gr::hier_block2
     ~fm_deemph();
 
     void set_tau(double tau);
+    void set_rate(double rate);
```

Comment:
`rate` argument should be a float, given that `d_quad_rate` is a float. Otherwise, double promotion warnings will result.

Reply:
Fixed.
Thanks.